### ISSUE TITLE
Large popups show scroll buttons although they shouldn't

### ISSUE THREAD

**Issue by fmeyertoens** [muckSponge/MaterialFox#230]
**Sanity checks (must complete)**
- [x] I have read and followed the installation instructions in the README
- [x] I have not modified the userChrome.css file
- [x] I have tested the latest release for my Firefox version, or commit on master/beta branch (beta is for Firefox Beta only)

**Describe the bug**
Large vertical popups show scroll buttons at the top and bottom by default in Firefox. MaterialFox tries to hide these:
https://github.com/muckSponge/MaterialFox/blob/3eb3971275ae33cd783373cb23473e81d1a6bf88/chrome/popup/popup.css#L177-L180
The `.menupopup-scrollbutton` selector doesn't seem to apply anymore. In fact I couldn't find an element with this class at all but the selector is used in various other places like `global.css` and `tabbar.css`.
As the two buttons don't have a class anymore, I suggest using their ID selectors `#scrollbutton-up, #scrollbutton-down`.

**To Reproduce**
1. Make a folder with many entries in the bookmarks toolbar.
2. A scrollbar and two scrollbuttons appear.

**System info**
 - OS: Windows 10
 - Firefox version: 76

**Comment by fmeyertoens** [comment]
Apparently `#scrollbutton-up` and `#scrollbutton-down` also apply to the tabbar scrollbuttons, so this is not ideal.

**Comment by muckSponge** [comment]
Yeah this kinda sucks. Part selector is not supported in userChrome.css. I've used a CSS variable inheritance workaround which assumes _all_ scrollbuttons can have a default `visibility` set to `visible` and the parent element (in this case, `menupopup`) overrides this with a CSS variable. It's not very readable or maintainable but it should work.
